marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/cloud/aws/AWSConfiguration.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2012-2020 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.central.cloud.aws;
+
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder;
+import com.amazonaws.services.simplesystemsmanagement.model.GetParameterRequest;
+import com.marklogic.hub.central.web.SslUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.web.ServerProperties;
+import org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory;
+import org.springframework.boot.web.server.WebServerFactoryCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import java.io.InputStream;
+
+@Configuration
+@Profile("aws")
+public class AWSConfiguration {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AWSConfiguration.class);
```

Comment:
For consistency, just extend LoggingObject, which defines this for you.

Reply:
I was going to extend `LoggingObject` but didn't since we are basically resuing functionality and so did not prefer to inherit.

Reply:
Okay - can you use the variable name "logger" then? Just wanted to avoid future questions of "Why is it logger in some places but LOGGER here?"